macros/parserDifferenceQuotient.pl

```diff
@@ -15,28 +15,27 @@
 ################################################################################
 
 =head1 NAME
-
 parserDifferenceQuotient.pl - An answer checker for difference quotients.
-
 =head1 DESCRIPTION
-
```

Comment:
Yes, I'd be happy to do that.  Those blank lines must have been munched in the process of copying and pasting from one text editor to another, or inadvertently by me.